accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -1085,13 +1088,13 @@ as:
  - `NNBD_TOP_MERGE(Object?, Object?)  = Object?`
  - `NNBD_TOP_MERGE(dynamic, dynamic)  = dynamic`
  - `NNBD_TOP_MERGE(void, void)  = void`
- - `NNBD_TOP_MERGE(Object?, void)  = void`
+ - `NNBD_TOP_MERGE(Object?, void)  = Object?`
```

Comment:
You generally use precedence, so would it be enough to write
>  - `NNBD_TOP_MERGE(_, _) = Object?`  
(perhaps with an "otherwise" attached).

Not necessary, just perhaps easier to understand.